### ISSUE TITLE
Microphone support

### ISSUE THREAD

**Issue by bryankeller** [alvr-org/alvr-visionos#102]
Apologies if I just missed a setting, but I see no way to use the headset microphone as the source mic in SteamVR. Is this possible today? Or is this an area that we'd like to tackle but we just haven't gotten to it yet?

**Comment by shinyquagsire23** [comment]
Yeah it's supported, it does require the driver mentioned in the initial setup though (and enabling the microphone setting)
